sale_validity/models/res_config_settings.py

```diff
@@ -0,0 +1,12 @@
+# © 2017 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    default_sale_order_validity_days = fields.Integer(
```

Comment:
Using default at the beginning on the name causes: 
![image](https://user-images.githubusercontent.com/44768500/59929529-49981700-9441-11e9-93d2-453899058981.png)

It also needs the parameter **readonly=False** to be able to be modified

```
sale_order_validity_days = fields.Integer(
        related='company_id.default_sale_order_validity_days',
        readonly=False,
    )
```